app/assets/stylesheets/white_theme/variables.scss

```
@@ -30,10 +30,33 @@ $two-column-narrow: 'only screen and (min-width: 749px) and (max-width: 920px)';
 $one-column: 'only screen and (max-width: 748px)';
 
 // fonts
+@font-face {
+  font-family: 'Alright-v2-Normal-Regular';
```

Maybe we should move away from an individual weight in the name:

```suggestion
  font-family: 'Alright-v2-Normal';
```

---

@ofsound thoughts?